.gitignore

```diff
@@ -72,4 +72,8 @@ geopandas/datasets/ne_110m_admin_0_countries.zip
 doc/source/getting_started/my_file.geojson
 
 .ruff_cache
-.env
\ No newline at end of file
+.env# pixi environments
```

Comment:
```suggestion
.env
# pixi environments
```

Reply:
If you are interested,  I can share the steps to get a development environment setup with pixi. Having dependencies that dependent on geopandas itself require a different approach (from the "standard") for the editable install of geopandas.

Reply:
I have it myself :).